MiniBlog/Services/UserService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MiniBlog.Model;
+using MiniBlog.Stores;
+
+namespace MiniBlog.Services
+{
+    public class UserService
+    {
+        private readonly IUserStore userStore;
```

Comment:
userStore should be encapsured in user service, shouldn't visaible in article service